modules/social_features/social_embed/social_embed.post_update.php

```diff
@@ -8,16 +8,22 @@
 use Drupal\Core\Site\Settings;
 
 /**
- * Populate the default value in new field for already existing users.
+ * Empty post update hook.
  */
 function social_embed_post_update_11001_populate_field_embed_content_settings(array &$sandbox): void {
+  // Moved to 11002.
```

Comment:
@navneet0693  I think we can't move this functionality to a new post update. There can be sites that already have embed `field_user_embed_content_consent` and some users have already could change that setting. So, if we will move this to a new post update all changed settings will be rewritten.

Let me know if I am mistaken.

Reply:
You are right, I will refactor the code to consider that situation.

Reply:
@Ressinel Please review the latest changes.